src/main/java/net/minecraftforge/fml/common/network/simpleimpl/SimpleNetworkWrapper.java

```diff
@@ -237,6 +238,25 @@ private String generateName(ChannelPipeline pipeline, ChannelHandler handler)
     public void sendToAll(IMessage message)
     {
         channels.get(Side.SERVER).attr(FMLOutboundHandler.FML_MESSAGETARGET).set(FMLOutboundHandler.OutboundTarget.ALL);
+        channels.get(Side.SERVER).attr(FMLOutboundHandler.FML_MESSAGETARGETEXCEPT).set(null);
+        channels.get(Side.SERVER).writeAndFlush(message).addListener(ChannelFutureListener.FIRE_EXCEPTION_ON_FAILURE);
+    }
+
+    /**
+     * Send this message to everyone except one player.
+     * The {@link IMessageHandler} for this message type should be on the CLIENT side.
+     *
+     * This is particularly useful for player-caused events that require instant feedback. The excepted player will not
+     * receive the message, and is therefore expected to reliably predict the outcome of the event.
+     * Must be used with care when synchronizing game logic.
```

Comment:
A use case where it may result in game logic desynchronization was explicitly mentioned because it seems the disclaimer only applies to the use cases I have mentioned. Someone may be using the method for other purposes, it is what it says on the tin after all.